app/models/cart.rb

```diff
@@ -29,7 +29,7 @@ def delete_accessory(accessory)
   def total
     @contents.map do |accessory_id, count|
       Accessory.find(accessory_id).price.to_f * count
-    end.sum
+    end.sum.round(2)
```

Comment:
We'll probably want a number to currency here... I'm happy to implement since I'm in cart land.